lib/tbd/geo.rb

```diff
@@ -693,6 +693,54 @@ def convex?(s1 = nil, s2 = nil)
     false
   end
 
+  ##
+  # Purge existing KIVA-related objects in an OpenStudio model. Resets ground-
+  # facing surface outside boundary condition to "Ground" or "Foundation".
+  #
+  # @param model [OpenStudio::Model::Model] a model
+  # @param boundary ["Ground", "Foundation"] new outside boundary condition
+  #
+  # @return [Bool] true if model is free of KIVA-related objects
+  # @retrun [false] if invalid input (see logs)
+  def resetKIVA(model = nil, boundary = "Foundation")
```

Comment:
In response to KIVA-related enhancement [request](https://github.com/rd2/tbd/issues/110) (Part 1). Part 2 will involve adding/testing TBD argument hash/measure option.